deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Zookeeper/config.xml

```diff
@@ -76,6 +76,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
+        <hudson.model.StringParameterDefinition>
+          <name>remote</name>
+          <description></description>
+          <defaultValue>processing-cluster-zookeepers</defaultValue>
```

Comment:
whats is this default? @Kaali09

Reply:
We have variablised the hostname in the zookeeper provision playbook.yml. So default is processing-cluster-kafka.